kroxylicious-filters/kroxylicious-record-encryption/etc/module-layering.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<!DOCTYPE import-control PUBLIC
+        "-//Checkstyle//DTD ImportControl Configuration 1.4//EN"
+        "https://checkstyle.org/dtds/import_control_1_4.dtd">
+
+<import-control pkg="io.kroxylicious.filter.encryption">
+    <!-- Allowed globally -->
+    <allow pkg="javax?\..*" regex="true"/>
+    <allow pkg="edu.umd.cs.findbugs.annotations"/>
+    <allow pkg="org.slf4j"/>
+    <allow pkg="io.micrometer.core.instrument"/>
+    <allow pkg="io.kroxylicious.proxy.tag"/>
+    <allow pkg="io.kroxylicious.filter.encryption.common"/>
+
+    <!-- top level package (these are all local-only=true)  -->
+    <allow pkg="org.apache.kafka.common.message" local-only="true"/>
+    <allow pkg="org.apache.kafka.common.record" local-only="true"/>
+    <allow pkg="io.kroxylicious.proxy.filter" local-only="true"/> <!-- the filter api -->
+    <allow pkg="io.kroxylicious.proxy.plugin" local-only="true"/> <!-- the plugin api -->
+    <allow pkg="io.kroxylicious.kms.service" local-only="true"/> <!-- the kms api -->
+    <allow pkg="io.kroxylicious.filter.encryption" regex="true" local-only="true"/><!-- all subpackages -->
+
+    <subpackage name="common">
```

Comment:
toself: i haven't figured out what the subpackage name refers to.

Reply:
It's a new package in the encryption filter.

Reply:
ah, got it.